### ISSUE TITLE
NPE in constructor of DefaultHelpFormatter 

### ISSUE THREAD

**Issue by wadoon** [xenomachina/kotlin-argparser#19]
Following line: 

```kotlin
    var hf = DefaultHelpFormatter()
```

leads to a NullPointerException:

```
Exception in thread "main" kotlin.KotlinNullPointerException
	at kotlin.coroutines.experimental.SequenceBuilderIterator.yield(SequenceBuilder.kt:163)
	at com.xenomachina.text.TextKt$codePointSequence$1.doResume(Text.kt:33)
	at kotlin.coroutines.experimental.jvm.internal.CoroutineImpl.resume(CoroutineImpl.kt:54)
	at kotlin.coroutines.experimental.SequenceBuilderIterator.hasNext(SequenceBuilder.kt:128)
	at com.xenomachina.text.term.TextTermKt.codePointWidth(TextTerm.kt:180)
	at com.xenomachina.argparser.DefaultHelpFormatter.<init>(DefaultHelpFormatter.kt:66)
	at edu.kit.iti.formal.automation.rvt.HelloKt.main(Hello.kt:16)
```

* kotlin-argparser version:  `com.xenomachina : kotlin-argparser : 2.0.1`
  * version `2.0.0` also effected
  * version `1.1.0` not effected
* Kotlin version: `1.1.2-4`

**Comment by xenomachina** [comment]
Can you try this with Kotlin version: 1.1.2-5? I am getting NPEs with 1.1.2-4 that I'm not seeing with 1.1.2-3 or 1.1.2-5, which makes me suspect that 1.1.2-4 has some weird bug in `SequenceBuilderIterator`.

**Comment by xenomachina** [comment]
I've changed xenocom (the underlying lib that used SequenceBuilder) to not use SequenceBuilder anymore, and this seems to stop the NPEs with 1.1.2-4. I should have a release of kotlin-argparser with this fix out shortly, but if you want to confirm that it solves the issue for you, you can temporarily add a direct dependency to xenocom 0.0.5.

**Comment by xenomachina** [comment]
I just released kotlin-argparser 2.0.2, which I believe fixes this issue.

If you are still seeing this problem after upgrading to that release, then please reopen this issue.

Thanks for the report, and especially for including the version numbers!
